content/en/docs/concepts/data-sources.md

```diff
@@ -126,3 +126,17 @@ For more information, see the [logs
 specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#log-signal),
 which covers topics including: log, defined fields, trace context fields and
 severity fields.
+
+## Baggage
+
+In addition to trace propagation, OpenTelemetry provides a simple mechanism for propagating name/value pairs, called `Baggage`. `Baggage` is intended for indexing observability events in one service with attributes provided by a prior service in the same transaction. This helps to establish a causal relationship between these events.
```

Comment:
Why use code font in this new section? E.g., this paragraph could read:

> ... called **baggage**. Baggage is ...

We can continue in the manner in the next paragraph:

> While baggage can ...

Reply:
I've used the formatting I saw int he original snippet this is from. Happy to edit it!